packages/api/tests/shared.ts

```diff
@@ -0,0 +1,24 @@
+import { VirtualPath, container } from '@sapphire/pieces';
+import { Route, RouterBranch } from '../src';
+
+export function makeRoute(route: string) {
+	// @ts-expect-error Stub
+	container.server ??= { options: {} };
+
+	class UserRoute extends Route {
+		public constructor(context: Route.LoaderContext) {
+			super(context, { route, methods: ['GET'] });
+		}
+
+		public override run() {
+			// nop
```

Comment:
```suggestion
			// noop
```

Super nitpick I know 😄